starfish/models/squid_model.py

```diff
@@ -39,6 +49,27 @@ def __init__(self, ocean, options=None):
 
         self._squid_ocean = self.get_squid_ocean()
 
+        # to get past codacy static method 'register_agent'
+        self._keeper = Keeper.get_instance()
+
+
+    def register_agent(self, service_name, endpoint_url, account, did=None):
```

Comment:
Does a consumer first create a squid_model, and then register a squid agent? Seems a bit roundabout.

Reply:
I think these deltas involving squid are orthogonal to the surfer test.

Reply:
No, the consumer does not touch any 'models' since it's the middleware, only stuff within the starfish library. There is already an ocean.register_agent exposed to the client.